src/utils.ts

```diff
@@ -35,7 +37,11 @@ export const writePact = (
     })
 }
 
-const constructInteraction = (intercept: Interception | AnyObject, testTitle: string): Interaction => {
+export const omitHeaders = (headers: AnyObject, blocklist: string[]) => {
+  return omit(headers, [...blocklist, ...AUTOGEN_HEADER_BLOCKLIST])
```

Comment:
Does this mean that if somebody wanted to exclude one of the items in `AUTOGEN_HEADER_BLOCKLIST` that they can't?

Reply:
I was thinking as part of msw-pact to provide with the package an exported array of default headers for exclusion, which the user could choose to add into their blocklist, or have a boolean to block the default list.

something like

```
import { cypressAutogenHeaderBlocklist } from 'cypress-pact'
{
    ...otherCypressConfig,
    "env": {
        'headersBlocklist': [...cypressAutogenHeaderBlocklist ]
        'headersBlockCypressAutogenerated': true
    }
}
```

or

```
{
    ...otherCypressConfig,
    "env": {
        'headersBlocklist': ['ignore-me-globally']
        'headersBlockCypressAutogenerated': true
    }
}
```

Reply:
@mefellows yes, the current implementation can't allow user to exclude item from `AUTOGEN_HEADER_BLOCKLIST`. 
We can introduce another environment variable (similar to @YOU54F 's suggestion) to allow user override default list.